apps/workflows-dashboard/public/config.js

```diff
@@ -0,0 +1,6 @@
+globalThis.env = {
```

Comment:
please check how this operates how this works if there is a change in .env

Reply:
We will deal only with env variables and therefore .env willnot be used for consideration